website/docs/cloud-docs/vcs/github.mdx

```diff
@@ -10,6 +10,10 @@ These instructions are for using GitHub.com for Terraform Cloud's VCS features,
 For beginning users on Terraform Cloud, we recommend using our [configuration-free GitHub App](/cloud-docs/vcs/github-app) to access repositories instead.
 <!-- END: TFC:only -->
 
+<!-- BEGIN: TFE:only -->
```

Comment:
Hey @mjyocca these comments only _exclude TFC content from TFE_ after content is copied over to the internal TFE repository, and not the other way around unfortunately. (not sure if you're aware of this process, but feel free to ask about it)
- additionally,`TFE:only` is not actually a valid content-exclusion comment so this will not work as you'd expect (assuming that is to hide TFE-only content from https://developer.hashicorp.com/terraform/cloud-docs/vcs/github).

I'd recommend removing these comments and having this content available on the `cloud-docs` content.

Reply:
@thiskevinwang was not aware the blocks were stripped entirely but good to know. Will remove 👍